docker-compose.yaml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+
+services:
+  owncast:
+    image: gabekangas/owncast:0.0.2
+    restart: unless-stopped
+    volumes:
+      - ${PWD}/config-example.yaml:/app/config.yaml # Adjust once you create your own config.yaml
+      - owncast_db:/db # Remove if you don't want chat persistant
+    command: /app/owncast --configFile=config.yaml --chatDatabase=/db/chat.db
+    ports:
+    - 8080:8080
+    - 1935:1935
+    labels:
+      - "traefik.enable=false"
+      - "traefik.http.routers.owncast.rule=Host(`live.your.org`)"
```

Comment:
Do you think it might be worthwhile to support environment variables for your hostname and email address so they don't have to edit the docker-config file itself?